code/modules/reagents/machinery/grinder.dm

```diff
@@ -436,7 +436,7 @@
 	if(istype(I, /obj/item/reagent_containers))
 		var/obj/item/reagent_containers/container = I
 		if(!container.standard_pour_into(user, src)) . = ..()
-	else if (LAZYLEN(I.reagents)) . = ..()
+	else if (I.reagents && I.reagents.total_volume) . = ..()
```

Comment:
```suggestion
	else if (I.reagents?.total_volume) . = ..()
```
This way if reagents is null then it won't cause a runtime and just return null, which is the same as the false & the previous I.reagents check.